src/services/storage/index.ts

```diff
@@ -189,6 +195,7 @@ const storage: CachedService = {
         confirmationService.removeAllListeners()
         stakingEventsEmitter.stop()
         storageManagerEventsEmitter.stop()
+        commsStop()
```

Comment:
This should be called on the `app.stop()` level as well.

Reply:
Done